docker-app/qfieldcloud/core/admin.py

```diff
@@ -924,6 +924,7 @@ class JobAdmin(QFieldCloudModelAdmin):
         "type",
         "status",
         "error_type",
+        "qgis_app_version",
```

Comment:
I don't think we need this info at this level. But we can show it as a readonly property when the Job form is opened.